CPAC/anat_preproc/lesion_preproc.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+
+from nipype.interfaces import afni
+import nipype.pipeline.engine as pe
+import nipype.interfaces.utility as util
+
+
+def inverse_lesion(lesion_path):
+    """
+    Check if the image contains more zeros than non-zeros, if so,
+    replaces non-zeros by zeros and zeros by ones.
+    It also copies the file to the working_dir and
+    Parameters
+    ----------
+    lesion_path: str
+        path to the nifti file to be checked and inverted if needed.
+    working_dir: str
+        path to the working directory
+
+    Returns
+    -------
+    new_path: str
+        path to the output file
+    """
+    import shutil
+    import os
+    import ntpath
+
+    import CPAC.utils.nifti_utils as nu
+    import nibabel as nib
+
+    new_path = os.path.join(os.getcwd(), ntpath.basename(lesion_path))
+
+    shutil.copyfile(lesion_path, new_path)
```

Comment:
I would not recommend copying the original, since it will just duplicate data.
To avoid it:
```
if has more zeros:
    new_path = ...
    nib.save(nii, new_path)
    return new_path

return lesion_path
```

Reply:
Ok I can remove that. I just thought as it is a separated function, I was afraid that it could be used outside of lesion_preproc and I wanted to ensure that it would not be possible to modify the source file directly after this function. 

Because you could do 
`path = inverse_lesion(lesion_path)`
And use path to do whatever modification and save it in path which modifies the source. 

But if you think I am paranoid I can change that :D

Reply:
Oh, I see, so thinking more of about usage outside C-PAC pipeline, right?

Ideally, if the function will change the data, it should create a new file to ensure immutability. I don't think you should worry much about this. You can add this information on the docstring, to make sure other devs will not modify the originals.